apps/etl-func/src/adapters/avro.ts

```diff
@@ -31,11 +31,6 @@ export const messageSchema = avro.Type.forSchema({
       name: "sender_user_id",
       type: "string",
     },
-    {
```

Comment:
We should change the apps/etl-func/avro/message.avsc too.

Should we remove it from apps/etl-func/src/__mocks__/message-event.ts ?